src/test/resources/net/sf/jabref/logic/exporter/MsBibExportFormatTest1.xml

```diff
@@ -1,67 +1,34 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <b:Sources xmlns:b="http://schemas.openxmlformats.org/officeDocument/2006/bibliography" xmlns="http://schemas.openxmlformats.org/officeDocument/2006/bibliography" SelectedStyle="">
 <b:Source>
-<b:SourceType>Report</b:SourceType>
```

Comment:
Why is this test file changed? Why isn't there generated a new file? Was the old file an invalid MsBib file?